news/2024/2024-06-03-the-followpoint-eriha-top-drum-basher.md

```diff
@@ -0,0 +1,102 @@
+
+*Who would you call the best osu!taiko player(s) at the moment and why?*
+
+> The best player I can think of right now is [SilverTyrant](https://osu.ppy.sh/users/14356353). He also competed in TWC2024 and I think it's very cool that he has recorded great scores on various maps, not just speed maps.
+>
+> Especially [Kage no Nai Hito](https://osu.ppy.sh/beatmapsets/1519045#taiko/3110395)'s [DT record](https://osu.ppy.sh/scores/1876120539) is so crazy that no player will surpass his record anytime soon.
+>
+> In addition, [sourufire](https://osu.ppy.sh/users/13249305) is probably the most growing of the Japanese players. His acc on the various speed maps is excellent and I look forward to competing with him in the future!
+
+*What do you feel is the most important aspect to keep motivation even when you are not improving, and what keeps you specifically motivated to play?*
+
+> I make it a point to take a break when my motivation wanes and observe the records of other players. Seeing other players happy to score well often motivates me naturally.
+>
+> I also make it a point to play other games and create cool down time. Lately I have been playing VALORANT with friends. I'm not good at fps, so I always lose lol.
+
+*Aside from osu!, do you play any other rhythm games?*
+
+> Recently I have been playing Sound Voltex. I'm by no means a good player, but I try to enjoy playing!
+>
+> If you know of any rhythm games you would recommend, please feel free to let me know!
+
+*Who would you want to meet IRL the most if you could meet any osu! player and why?*
+
+> There are many players I would like to meet, but I would really like to meet [Yuemiao](https://osu.ppy.sh/users/4493348) and [hz404](https://osu.ppy.sh/users/14947043), who are irreplaceable for me!
+>
+> Also, I would like to meet [Ney](https://osu.ppy.sh/users/5991961) and [uone](https://osu.ppy.sh/users/5321719) who I respect for their playing style!
+
+*Do the communities of Taiko no Tatsujin and osu!taiko overlap in the pro player scene in Japan, and do arcade pro players play osu!taiko too?*
+
+> Although it cannot be said that good play in Taiko no Tatsujin is proportional to good play in osu!taiko, the top players in Taiko no Tatsujin are often the top players in osu!taiko!
+>
+> For example, players such as [_yu68](https://osu.ppy.sh/users/6170507), [1RoHa_](https://osu.ppy.sh/users/4632503), [kotohira_06](https://osu.ppy.sh/users/22444525), and [supesan](https://osu.ppy.sh/users/14623658) are both top players in Taiko no Tatsujin and in osu!taiko!
+
+*To finish things up, what does the future hold for you? Do you have any goals or anything else that you'd like to mention?*
+
+> I would like to keep playing this game for fun first, and I also have a goal to fill all my BPs with more than 900pp.
```

Comment:
im not sure I've ever seen anyone refer to their top plays as "BPs", although I do get what it means. maybe it should be reworded to be a bit more clear on what they meant by it?